jenkins/workshop/content/lab17.md

```diff
@@ -60,7 +60,7 @@ In your pipeline, add the Vulnerability Scan Stage after the Build Image Stage.
     stage('Clair Container Vulnerability Scan') {
       steps {
             sh "oc login -u $ocuser -p $ocpass --insecure-skip-tls-verify https://$ocp 2>&1"
-            sh 'skopeo --debug copy --src-creds="$(oc whoami)":"$(oc whoami -t)" --src-tls-verify=false --dest-tls-verify=false' + " --dest-creds=$quayuser:$quaypass docker://docker-registry.default.svc:5000/cicd-$ocuser/jboss-eap70-openshift:1.5 docker://quay-enterprise-quay-enterprise.apps.$ocp/$quayuser/$quayrepo:1.5"
+            sh 'skopeo --debug copy --src-creds="$(oc whoami)":"$(oc whoami -t)" --src-tls-verify=false --dest-tls-verify=false' + " --dest-creds=$quayuser:$quaypass docker://docker-registry.default.svc:5000/cicd-$ocuser/jboss-eap70-openshift:1.5 docker://quay.apps.$ocp/$quayuser/$quayrepo:1.5"
         }
```

Comment:
This one uses the route, not the service

Reply:
I think I saw this one first and then my mind was set on that for all of the others...